sdcm/scylla_bench_thread.py

```diff
@@ -78,30 +79,40 @@ class ScyllaBenchThread(DockerBasedStressThread):  # pylint: disable=too-many-in
     DOCKER_IMAGE_PARAM_NAME = "stress_image.scylla-bench"
     _SB_STATS_MAPPING = {
         # Mapping for scylla-bench statistic and configuration keys to db stats keys
-        'Mode': 'Mode',
-        'Workload': 'Workload',
-        'Timeout': 'Timeout',
-        'Consistency level': 'Consistency level',
-        'Partition count': 'Partition count',
-        'Clustering rows': 'Clustering rows',
-        'Page size': 'Page size',
-        'Concurrency': 'Concurrency',
-        'Connections': 'Connections',
-        'Maximum rate': 'Maximum rate',
-        'Client compression': 'Client compression',
-        'Clustering row size': 'Clustering row size',
-        'Rows per request': 'Rows per request',
-        'Total rows': 'Total rows',
-        'max': 'latency max',
-        '99.9th': 'latency 99.9th percentile',
-        '99th': 'latency 99th percentile',
-        '95th': 'latency 95th percentile',
-        '90th': '90th',
-        'median': 'latency median',
-        'Operations/s': 'op rate',
-        'Rows/s': 'row rate',
-        'Total ops': 'Total partitions',
-        'Time (avg)': 'Total operation time',
+        'Mode': (str, 'Mode'),
+        'Workload': (str, 'Workload'),
+        'Timeout': (int, 'Timeout'),
```

Comment:
Have you tested it?

Timeout may have letters like in the `1m` value. How it should be translated to an `int`eger?

Reply:
convert_metric_to_ms seems to be supporting it.

Worst case it would be translated to string, currently no code is reading it